plugin/notmuch_abook.vim

```diff
@@ -67,5 +67,7 @@ augroup notmuchabook
     au!
     au FileType mail call InitAddressBook()
     au FileType mail setlocal completefunc=CompleteAddressBook
+    au FileType notmuch-compose call InitAddressBook()
+    au FileType notmuch-compose setlocal completefunc=CompleteAddressBook
```

Comment:
I believe `au FileType mail,notmuch-compose …` can achieve the same thing avoiding code duplication. Could you try and update your PR? :-)